framework/CodeInterfaces/AccelerateCFD/AccelerateCFD.py

```diff
@@ -0,0 +1,356 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on August 31, 2020
+
+@author: Andrea Alfonsi
+
+comments: Interface for AccelerateCFD
+"""
+import os
+import numpy as np
+import pandas as pd
+import xml.etree.ElementTree as ET
+import glob
+import math
+from sklearn import neighbors
+from OpenFoamPP import fieldParser
+from CodeInterfaceBaseClass import CodeInterfaceBase
+from GenericCodeInterface import GenericParser
+
+def findNearest(array, value):
```

Comment:
I think we have a similar method in mathUtils, if not, should we move it there?

Reply:
I have updated the mathUtils method to handle this.